bentoml/_internal/server/__init__.py

```diff
@@ -131,6 +191,30 @@ def serve_production(
 ) -> None:
     working_dir = os.path.realpath(os.path.expanduser(working_dir))
     svc = load(bento_identifier, working_dir=working_dir)
+    serve_info = get_serve_info()
+
+    # Track first time
+    if svc.bento is not None:
+        bento = svc.bento
+        event_properties = BentoServeProductionOnStartupEvent(
+            serve_id=serve_info.serve_id,
+            serve_creation_timestamp=serve_info.serve_creation_timestamp,
+            bento_identifier=bento_identifier,
+            bento_creation_timestamp=bento.info.creation_time,
+            model_tags=bento.info.models,
+            model_types=[
+                bento._model_store.get(i).info.module for i in bento.info.models
+            ],
+        )
+    else:
+        # In this case serving from a file/directory with --production
+        event_properties = BentoServeProductionOnStartupEvent(
+            serve_id=serve_info.serve_id,
+            serve_creation_timestamp=serve_info.serve_creation_timestamp,
+            bento_identifier=bento_identifier,
```

Comment:
remove `bento_identifier` here and above - it contains user file name, or bento name

Reply:
done